loader/vk_loader_platform.h

```diff
@@ -353,7 +353,14 @@ static inline char *loader_platform_executable_path(char *buffer, size_t size) {
 
     return buffer;
 }
-#elif defined(__Fuchsia__) || defined(__OpenBSD__)
+#elif defined(__Fuchsia__)
+static inline char *loader_platform_executable_path(char *buffer, size_t size) {
+    if (size < 1) return NULL;
+    // Fuchsia doesn't support getting the executable path. The blank path should match no layers.
+    buffer[0] = '\0';
+    return buffer;
```

Comment:
```suggestion
    (void)buffer;
    (void)size;
    return NULL;
```

loader_platform_executable_path should just return NULL so that no client code will "think" it is valid. That is what the current fuschia code does, so should just keep doing that.

Happy to discuss alternative solutions if returning NULL isn't enough to get the loader working on fuschia.